_inc/client/plans/product-selector.jsx

```diff
@@ -0,0 +1,195 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { moment, translate as __ } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ProductCard from '../components/product-card';
+import { SingleProductBackup } from './single-product-backup';
+import { getPlanClass } from '../lib/plans/constants';
+import { getActiveSitePurchases, getSitePlan, isFetchingSiteData } from '../state/site';
+import { getSiteRawUrl, getUpgradeUrl } from '../state/initial-state';
+import { getProducts, isFetchingProducts } from '../state/products';
+// import { JETPACK_PRODUCTS } from '../lib/products/constants';
+
+class ProductSelector extends Component {
+	getTitleSection() {
+		return (
+			<Fragment>
+				<h1 className="plans-section__header">{ __( 'Solutions' ) }</h1>
+				<h2 className="plans-section__subheader">
+					{ __( "Just looking for backups? We've got you covered." ) }
+				</h2>
+			</Fragment>
+		);
+	}
+
+	getProductCardPropsForPurchase( purchase ) {
+		const { siteRawlUrl } = this.props;
+
+		const planClass = getPlanClass( purchase.product_slug );
+
+		const dailyBackupTitle = __( 'Jetpack Backup {{em}}Daily{{/em}}', {
```

Comment:
Would be really nice if all those titles, descriptions, product slugs, etc. are defined somewhere else, outside of the components. That would contribute to a nice separation of concerns.

Reply:
Great suggestion!

Reply:
Adding a TODO for now. Can be addressed separately.